### PR TITLE
Exclude some common large directories in ChokidarFileWatchProvider

### PR DESCRIPTION
**Pull request by adampauls**
Temporary and hopefully inoffensive fix for https://github.com/microsoft/pyright/issues/5415. It would be better to pipe through `configOptions.exclude` but that's al arger change.

### CODE REVIEW

File: packages/pyright-internal/src/common/chokidarFileWatcherProvider.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class ChokidarFileWatcherProvider implements FileWatcherProvider {
             watcherOptions.usePolling = false;
         }
 
-        const excludes: string[] = ['**/__pycache__/**'];
+        const excludes: string[] = ['**/node_modules/**', '**/__pycache__/**', '**/.*/**'];
```
Comment (rchiodo) [inline packages/pyright-internal/src/common/chokidarFileWatcherProvider.ts:47]: Doesn't this exclude everything?

'**/.*/**'

Comment (rchiodo) [reply packages/pyright-internal/src/common/chokidarFileWatcherProvider.ts:47]: oh no, it excludes any `.` folders, like `.git`. That seems okay?

Comment (adampauls) [reply packages/pyright-internal/src/common/chokidarFileWatcherProvider.ts:47]: Yeah, I just copied from [here](https://github.com/microsoft/pyright/blob/0a67c65bef4ce95d4de8d2218a0bd875d2fb9230/packages/pyright-internal/src/analyzer/service.ts#L580)

**Comment by adampauls** [comment]
Better to fix https://github.com/RobertCraigie/pyright-python/issues/191

**Pull request closed without merging**